src/Veldrid.MetalBindings/Veldrid.MetalBindings.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.4.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.0" />

```

Comment:
I had to update some packages, due to mismatches with Vortice.Windows otherwise

Reply:
Seems fine to me.